go.mod

```diff
@@ -2,39 +2,25 @@ module github.com/NearNodeFlash/nnf-deploy
 
 go 1.19
 
-replace (
-	github.com/HewlettPackard/dws => ./dws
-	github.com/NearNodeFlash/lustre-fs-operator => ./lustre-fs-operator
-	github.com/NearNodeFlash/nnf-dm => ./nnf-dm
-	github.com/NearNodeFlash/nnf-sos => ./nnf-sos
-)
+replace github.com/HewlettPackard/dws => ./dws
```

Comment:
should dws be removed here too?

Reply:
main.go uses `dwsv1alpha2.SystemConfiguration{}`